vpdq/benchmark/README.MD

```diff
@@ -0,0 +1,63 @@
+# pytx-vpdq
+Benchmark vPDQ CPP implementation and python-binding library
+
+
+# Observed Performance
+MacBook Pro (16-inch, 2021), Apple M1 Pro, 32 GB Ram
+
+Results:
+-------
+```
+% python3 benchmark.py      
+Python hashing time: 34.1643s
+  Total hash:7069  Per hash: 4.8330ms
```

Comment:
nit: In some output you put a space between the ':' and the number, and sometimes you don't

Reply:
Good catch! Have changed it.